Makefile

```diff
@@ -7,3 +7,58 @@ generate_migration:	## Generate a new migration
 
 generate_api_clients:	## Generate API clients
 	go generate ./api-clients/convoy/generate.go
+	
+.PHONY: docker_services firebase_emulator launch_scenario ingest_data send_webhooks cron_scheduler workers reset_db
+
+# Default environment file
+ENV_FILE=.env
+
+## Start docker services
+docker_services:
+	@set -e; \
+	echo "Starting Docker services..."; \
+	docker compose up -d
+
+## Run Firebase emulator
+firebase_emulator: 
+	@echo "Starting Firebase emulator..."
+	set -a && source $(ENV_FILE) && firebase --project "$$GOOGLE_CLOUD_PROJECT" emulators:start --import=./firebase-local-data --export-on-exit
```

Comment:
Applicable to all commands. `set -a` prints the environment, which more often than not, will print secrets on the terminal. Let's not.

Reply:
Also, I thought we ended up on `mise` to manage our environment, why are we doing that again manually? Same thing goes for tasks itself, why are we using `make` instead of `mise`?